src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceImpl.java

```diff
@@ -0,0 +1,665 @@
+        }
+      }
+    );
+    classificationStrategy.put(SKAGIT_TRANSIT_AGENCY_ID, routeData -> RideType.SKAGIT_TRANSIT);
+    classificationStrategy.put(
+      SEATTLE_STREET_CAR_AGENCY_ID,
+      routeData -> RideType.SEATTLE_STREET_CAR
+    );
+    classificationStrategy.put(
+      WASHINGTON_STATE_FERRIES_AGENCY_ID,
+      routeData -> RideType.WASHINGTON_STATE_FERRIES
+    );
+    classificationStrategy.put(KITSAP_TRANSIT_AGENCY_ID, routeData -> RideType.KITSAP_TRANSIT);
+
+    // Spaces have been removed from the route name because of inconsistencies in the WSF GTFS route dataset.
+    washingtonStateFerriesFares.put(
+      "Seattle-BainbridgeIsland",
+      ImmutableMap.of(FareType.regular, 9.25f, FareType.senior, 4.60f)
+    );
+    washingtonStateFerriesFares.put(
+      "Seattle-Bremerton",
+      ImmutableMap.of(FareType.regular, 9.25f, FareType.senior, 4.60f)
+    );
+    washingtonStateFerriesFares.put(
+      "Mukilteo-Clinton",
+      ImmutableMap.of(FareType.regular, 5.65f, FareType.senior, 2.80f)
+    );
+    washingtonStateFerriesFares.put(
+      "Fauntleroy-VashonIsland",
+      ImmutableMap.of(FareType.regular, 6.10f, FareType.senior, 3.05f)
+    );
+    washingtonStateFerriesFares.put(
+      "Fauntleroy-Southworth",
+      ImmutableMap.of(FareType.regular, 7.20f, FareType.senior, 3.60f)
+    );
+    washingtonStateFerriesFares.put(
+      "Edmonds-Kingston",
+      ImmutableMap.of(FareType.regular, 9.25f, FareType.senior, 4.60f)
+    );
+    washingtonStateFerriesFares.put(
+      "PointDefiance-Tahlequah",
+      ImmutableMap.of(FareType.regular, 6.10f, FareType.senior, 3.05f)
+    );
+    washingtonStateFerriesFares.put(
+      "Anacortes-FridayHarbor",
+      ImmutableMap.of(FareType.regular, 14.85f, FareType.senior, 7.40f)
+    );
+    washingtonStateFerriesFares.put(
+      "Anacortes-LopezIsland",
+      ImmutableMap.of(FareType.regular, 14.85f, FareType.senior, 7.40f)
+    );
+    washingtonStateFerriesFares.put(
+      "Anacortes-OrcasIsland",
+      ImmutableMap.of(FareType.regular, 14.85f, FareType.senior, 7.40f)
+    );
+    washingtonStateFerriesFares.put(
+      "Anacortes-ShawIsland",
+      ImmutableMap.of(FareType.regular, 14.85f, FareType.senior, 7.40f)
+    );
+    washingtonStateFerriesFares.put(
+      "Coupeville-PortTownsend",
+      ImmutableMap.of(FareType.regular, 3.85f, FareType.senior, 1.90f)
+    );
+    washingtonStateFerriesFares.put(
+      "PortTownsend-Coupeville",
+      ImmutableMap.of(FareType.regular, 3.85f, FareType.senior, 1.90f)
+    );
+    washingtonStateFerriesFares.put(
+      "Southworth-VashonIsland",
+      ImmutableMap.of(FareType.regular, 6.10f, FareType.senior, 3.05f)
+    );
+
+    OrcaSoundTransitLinkFares.populateLinkFares(soundTransitLinkFares);
+    OrcaSoundTransitSounderFares.populateSounderFares(soundTransitSounderFares);
+  }
+
+  // If set to true, the test ride price is used instead of the actual agency cash fare.
+  public boolean IS_TEST;
```

Comment:
Can you please remove this and replace it with a subclass where this is overridden?

Example: https://github.com/opentripplanner/OpenTripPlanner/blob/10290ad1d83b8cf70181e91a5b10f21183646af2/src/ext-test/java/org/opentripplanner/ext/fares/impl/AtlantaFareServiceTest.java#L250-L268

Reply:
This is clean! Great suggestion